### ISSUE TITLE
ERR_NAME_NOT_RESOLVED

### ISSUE THREAD

**Issue by leonwong72** [silkimen/cordova-plugin-advanced-http#278]
I would like to ask why am I getting ERR_NAME_NOT_RESOLVED when logging in. I run my apps in browser and post to server through API. Below is screenshot of my code.
![2019-12-12 (1)](https://user-images.githubusercontent.com/42371049/70678826-29e5fb00-1cce-11ea-87f1-8f35d37632e4.png)

**Comment by silkimen** [comment]
Looks like your backend is not reachable. Could be a network problem. Please use [StackOverflow](https://stackoverflow.com/questions/tagged/cordova-plugin-advanced-http) for this kind of questions.

**Comment by leonwong72** [comment]
I've found out the problem for this error. But now I have another problem. When my project tries to post or get datas from API, it returns the following error.

> Access to XMLHttpRequest at [domain] from origin 'http://127.0.0.1:8081' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource

I googled about this error when I'm using AXIOS before and it says using this plugin can solve this problem, but it doesn't.
Below are the CSP and config.xml file and the usage of this plugin.
![Capture](https://user-images.githubusercontent.com/42371049/70878138-3c787100-1ffb-11ea-9b90-a7ff077fb428.PNG)
![Capture1](https://user-images.githubusercontent.com/42371049/70878139-3c787100-1ffb-11ea-9220-537475ecc5dc.PNG)
![Capture2](https://user-images.githubusercontent.com/42371049/70878140-3d110780-1ffb-11ea-92cb-e4a7f3ba036d.PNG)

**Comment by silkimen** [comment]
This is a new question. Please use [StackOverflow](https://stackoverflow.com/questions/tagged/cordova-plugin-advanced-http) for this kind of questions next time. This plugin will solve your CORS issue on iOS and Android, but not in the browser.  Closing this, feel free to open a new issue, if this won't solve your problem! 👍
